src/app/_components/GovernanceCalendarCards.tsx

```diff
@@ -0,0 +1,108 @@
+import { Heading } from '@/components/Heading'
+import { TextLink } from '@/components/TextLink'
+
+import { extractTimeFromISO } from '@/utils/convertISOTime'
+
+import { GOOGLE_CALENDAR_API_URL } from '@/constants/apiUrls'
+
+type CalendarProps = {
+  startDate: string
+}
+
+function Calendar({ startDate }: CalendarProps) {
+  const { day, month } = extractTimeFromISO(startDate)
+
+  return (
+    <div className="grid min-h-40 min-w-36 grid-rows-[_40px,auto] rounded-md border border-blue-400 bg-blue-500">
+      <span className="flex items-center justify-center rounded-t text-base font-normal uppercase">
+        {month}
+      </span>
+      <span className="flex flex-1 items-center justify-center rounded-b bg-blue-100 text-5xl font-bold text-blue-400">
+        {day}
+      </span>
+    </div>
+  )
+}
+
+const eventSchema = z.object({
+  id: z.string(),
+  start: z.object({ dateTime: z.string() }),
+  end: z.object({ dateTime: z.string() }),
+  htmlLink: z.string(),
+  summary: z.string(),
+})
+
+const eventsSchema = z.object({
+  items: z.array(eventSchema),
+})
+
+async function getEvents(endpoint: string) {
+  const response = await fetch(endpoint)
+
+  if (!response.ok) {
+    throw new Error('getEvents: Failed to fetch events')
+  }
+
+  const data = await response.json()
+  return eventsSchema.parse(data)
+}
+
+export function GovernanceCalendarCards() {
+  const [currentDate] = useState(new Date())
+  const timeMin = encodeURIComponent(currentDate.toISOString())
+
+  const url = `${GOOGLE_CALENDAR_API_URL}${process.env.NEXT_PUBLIC_CALENDAR_ID}/events?maxResults=6&singleEvents=true&timeMin=${timeMin}&key=${process.env.NEXT_PUBLIC_CALENDAR_API_KEY}`
+
+  const { data: events, error } = useSWR(url, getEvents)
+
+  if (error) return <div>Failed to load events</div>
+  if (!events) return <div>Loading events...</div>
+
+  return (
+    <CardLayout type="blogPost">
+      {events.items.map((event) => {
+        const { id, start, end, htmlLink, summary } = event
+
+        const { time: startTime } = extractTimeFromISO(start.dateTime)
+        const { time: endTime } = extractTimeFromISO(end.dateTime)
+
+        return (
+          <li
+            key={id}
+            className="flex flex-col rounded-md border border-blue-500 p-1 sm:flex-row"
+          >
+            <Calendar startDate={start.dateTime} />
+            <div className="flex flex-1 flex-col items-start justify-between gap-6 p-4">
+              <div className="flex flex-col gap-3">
+                <div className="flex gap-2">
+                  <Badge
+                    borderColor="brand-100"
+                    icon={Clock}
+                  >{`UTC ${startTime} - ${endTime}`}</Badge>
+                  <Badge>Zoom</Badge>
+                </div>
+                <Heading tag="h3" variant="lg">
+                  {summary}
+                </Heading>
+              </div>
+              <TextLink className="flex gap-2 text-base" href={htmlLink}>
```

Comment:
We don't want to use the `className` prop. We should actually create a ticket to remove it so we're not tempted.

Reply:
We create a separate one! Thank you.